### ISSUE TITLE
Link to current version from 7.8 broken (nesting, ../ missing)

### ISSUE THREAD

**Issue by Marwe** [OSGeo/grass-website#343]
At
[GRASS GIS 7.8 Addons Manual pages](https://grass.osgeo.org/grass78/manuals//addons/)
> Note: This addon documentation is for an older version of GRASS GIS that will be discontinued soon. You should upgrade your GRASS GIS installation, and read the [current addon manual page](https://grass.osgeo.org/grass78/grass82/manuals/addons/index.html).

Link falsely pointing to `grass.osgeo.org/grass78/grass82/manuals` via 
```html
<a href="../../../grass82/manuals/addons/index.html">current addon manual page</a>
```
* so the nesting does not fit (one more `../` would solve it)

Other similar info boxes in the same documentation work as expected.

**Comment by neteler** [comment]
Quick analysis: the problem seems to be a glitch in the red-box-injection when `index.html` isn't given:

- works ok: https://grass.osgeo.org/grass78/manuals/addons/index.html
- fails: https://grass.osgeo.org/grass78/manuals/addons/

**Comment by neteler** [comment]
Does @tmszi have an idea how to solve this?

**Comment by tmszi** [comment]
> Does @tmszi have an idea how to solve this?

Link https://grass.osgeo.org/grass78/manuals//addons/ contains 2  `/` between **manuals`//`addons** and Apache web server added this additional `/` according server config (check the Apache web server config or send me it privately please).

**current addon manual page** link path is right 
```html
<a href="../../../grass82/manuals/addons/index.html">current addon manual page</a>
```

If you try URL https://grass.osgeo.org/grass78/manuals/addons/  with one `/` between **manuals`/`addons** link **current addon manual page** loaded correctly.

**Comment by neteler** [comment]
Thanks to @tmszi the trailing slash situation in the Apache config has been updated.
I have now updated the server configuration with the modifications.

So, any wrong URL which contains 2 `/` between `manuals//addons` will not work.
Hence, the user needs to change `...//...` to single `/`. (Not sure if any Apache rule exists to rewrite that automatically.)

Importantly, hugo + Apache must not **generate** double slashes. Let's watch out if this still occurs after this config update.
